app/controllers/concerns/posts_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PostsController < ApplicationController
+  def index
+    @posts = @user.posts.includes(:comments).where(author_id: params[:user_id])
+    @user = User.find(params[:user_id])
+    @posts = @posts.paginate(page: params[:page], per_page: 3)
+  end
```

Comment:
- Good job implementing the changes from the previous reviewer, but your code throws an error, this is because you are trying to access the `@user` variable before declaring the variable. Please fix this 👍 👍 
```ruby
     @user = User.find(params[:user_id])
      @posts = @user.posts.includes(:comments)
```